web/src/lib/i18n/en.json

```diff
@@ -717,20 +792,37 @@
   "previous_or_next_photo": "Previous or next photo",
   "primary": "Primary",
   "profile_picture_set": "Profile picture set.",
+  "public_album": "Public album",
   "public_share": "Public Share",
+  "reaction_deleted": "{type, select, comment {Comment} like {Like} other {Reaction}} deleted",
```

Comment:
I don't think we should include mappings to an enum in the translation file. It has no type safety and it forces us to include `other` which often doesn't make sense for an enum.

Reply:
Understood, how would you fix that?
`reaction.type == Type.Comment ? $t('comment_deleted') : $t('like_deleted')`
Unfortunatelly there is always "else/other".

Reply:
Something like this should work

```ts
const messages: Record<Type, string> = {
  [Type.Comment]: $t('comment_deleted'),
  [Type.Like]: $t('like_deleted'),
};

messages[reaction.type];
```